lib/logcat.js

```diff
@@ -1,118 +1,80 @@
-"use strict";
+import { SubProcess } from 'teen_process';
+import log from './logger';
```

Comment:
hmm interesting. It is a choice that we have to make to have `ADB` as prefix or `LOGCAT` as prefix or as `ADB [LOGCAT ...` currently it is the third one.